pynest/nest/tests/test_sp/test_mpitests.py

```diff
@@ -19,45 +19,41 @@
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
-import unittest
-import nest
 import os
-from subprocess import call
+import subprocess
+import unittest
 
-HAVE_MPI = nest.ll_api.sli_func("statusdict/have_mpi ::")
+cmd = ["nest", "-c", "statusdict/have_mpi :: =only"]
+HAVE_MPI = subprocess.check_output(cmd) == "true"
```

Comment:
@jougs I think `subprocess.check_output()` returns data as encoded bytes for Python 3 and as regular strings for Python 2. The test will only run if I use Python 2 and when I use Python 3 I get the message `skipped 'NEST was compiled without MPI'`, even though I have compiled with MPI. The same is true for `test_connect_all_patterns`. The two tests are also skipped during `make installcheck`.